docs/guide/robot_sbc/setup_jetson_nano.md

```diff
@@ -66,76 +136,364 @@ pip3 install -U gdown
+
+These instructions work for Jetpack 4.6.1. 
+
+* If you have a 4gb Jetson Nano then download Jetpack 4.6.1 from Nvidia
+  here: [jetson-nano-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jeston_nano/jetson-nano-jp461-sd-card-image.zip)
+* If you have a 2gb Jetson Nano the download Jetpack 4.5.1from Nvidia
+  here: [jetson-nano-2gb-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jetson_nano_2gb/jetson-nano-2gb-jp461-sd-card-image.zip)
+
+
+Visit the official [Nvidia Jetson Nano Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-devkit#prepare)
+or [Nvidia Xavier NX Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-xavier-nx-devkit).
+Work through the __Prepare for Setup__, __Writing Image to the microSD Card__,
+and __Setup and First Boot__ instructions, then return here.
+
+Once you're done with the setup, ssh into your vehicle. Use the terminal for
+Ubuntu or Mac. [Putty](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html)
+for windows.
+
+
+Remove Libre Office:
 
 ```bash
-cd ~/projects
+sudo apt-get remove --purge libreoffice*
+sudo apt-get clean
+sudo apt-get autoremove
+```
+
+And add a 6GB swap file:
+
+```bash
+git clone https://github.com/JetsonHacksNano/installSwapfile
+cd installSwapfile
+./installSwapfile.sh
+reboot 
+```
+
+#### Step 2b: Free up the serial port (optional. Only needed if you're using the Robohat MM1)
+
+```bash
+sudo usermod -aG dialout <your username>
+sudo systemctl disable nvgetty
+```
+
+
+#### Step 3b: Setup Python Environment
+
+The installation on JP 4.6.X is more tricky, because we need to use a newer
+version of tensorflow than what is packaged by NVidia for this OS version.
+Also, we need a compatible OpenCV version which needs to be built on the nano.
+
+* Step 1: Install mamba-forge
+
+Download and install mambaforge
+
+```bash
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-aarch64.sh
+chmod u+x ./Mambaforge-Linux-aarch64.sh
+bash ./Mambaforge-Linux-aarch64.sh
 ```
 
-* Get the latest donkeycar from Github.
-> Note: There are currently version updates happening on the `main` branch so you might rather want to checkout the latest stable release as explained below.
+* Step 2: Install git lfs
 
 ```bash
+sudo apt-get install git-lfs
+```
+
+* Step 3: Download Donkey Car
+
+Downloading Donkey Car and the Tensorflow wheel for jp46 and tf29
+
+```bash
+mkdir projects
+cd projects
 git clone https://github.com/autorope/donkeycar
+git-lfs clone https://github.com/autorope/jetson
 cd donkeycar
 git checkout main
-pip3 install -e .[nano]
 ```
 
-* Alternatively, to get the latest stable release
+* Step 4: Create the python env and install donkey
+
+Also install tensorflow in the last step
+
+```bash
+mamba env create -f install/envs/jetson46.yml
```

Comment:
`-bash: mamba: command not found`. I rebooted to fix it but I think the correct fix is `source ~/.bashrc`

Reply:
Good point, I've added this to the end of the mamba forge install.